### DISCUSSION TITLE
Rename "quotient" variable from the "getWithdrawableAmount" method.

### DISCUSSION THREAD

**Discussion by andreivladbrg** [sablier-labs/evm-monorepo#93] [General]
The variable indeed is the quotient of `elapsedTime` and `totalTime`, but this does not say much about what it represents.

I'm proposing we change the variable name to something more specific, like `percentStreamed`.

Do you have any idea for a better name?

**Comment by PaulRBerg** [comment]
I agree that the `quotient` name is vague.

But I don't like `percentStreamed` either - that'd be a bit confusing from a lexical point of view because it's the money that gets streamed, not the percentage.

What about one of these?

- `timePercentage`
- `timeRatio`
- `timeQuotient`
- `elapsedTimePercentage`
- `elapsedTimeRatio`
- `elapsedTimeQuotient`

**Reply by PaulRBerg** [reply]
@andreivladbrg any new thoughts on this?

**Comment by PaulRBerg** [comment]
Locking, we ended up renaming `quotient` to` elapsedTimePercentage`.
